### PR TITLE
manual: minor rewording

### PR DESCRIPTION
**Pull request by createyourpersonalaccount**
# Checklist:
Everything in this checklist is required for each PR.  Please do not approve a PR that does not have all of these items.

- [X] I have pulled from master before submitting this PR
- [X] There are no merge conflicts.
- [X] I've added the new dependencies as:
  - [X] ASDF dependencies,
  - [X] Git submodules,
    ```sh
	cd /path/to/nyxt/checkout
    git submodule add https://gitlab.common-lisp.net/nyxt/py-configparser _build/py-configparser
    ```
  - [X] and Guix dependencies.
- [X] My code follows the style guidelines for Common Lisp code. See:
  - [Norvig & Pitman's Tutorial on Good Lisp Programming Style (PDF)](https://www.cs.umd.edu/~nau/cmsc421/norvig-lisp-style.pdf)
  - [Google Common Lisp Style Guide](https://google.github.io/styleguide/lispguide.xml)
- [X] I have performed a self-review of my own code.
- [x] My code has been reviewed by at least one peer.  (The peer review to approve a PR counts.  The reviewer must download and test the code.)
- [X] Documentation:
  - [X] All my code has docstrings and `:documentation`s written in the aforementioned style.  (It's OK to skip the docstring for really trivial parts.)
  - [X] I have updated the existing documentation to match my changes.
  - [X] I have commented my code in hard-to-understand areas.
  - [X] I have updated the `changelog.lisp` with my changes if it's anything user-facing (new features, important bug fix, compatibility breakage).
  - [X] I have added a `migration.lisp` entry for all compatibility-breaking changes.
- [X] Compilation and tests:
  - [X] My changes generate no new warnings.
  - [X] I have added tests that prove my fix is effective or that my feature works.  (If possible.)
  - [X] New and existing unit tests pass locally with my changes.

### CODE REVIEW

File: source/manual.lisp [hunk 1]
```diff
@@ -228,32 +228,31 @@ all hosts being blocked, execute command " (:code "describe-variable") ", choose
...
+conditions. URL-dispatchers can also be used for this, but it is
+simpler to use an auto-rule. Given that Nyxt's functionality is
+mode-based, the consequences are far reaching.")
         (:p "These can be used in the following ways:")
         (:ul
-         (:li "Manually, by calling "
-              (:nxref :command 'save-non-default-modes-for-future-visits)
-              " or " (:nxref :command 'save-exact-modes-for-future-visits) ".")
+         (:li "Manually, by calling:")
          (:ul
```
Comment (aartaka) [inline source/manual.lisp:242]: Here a `dl` HTML element would work even better

**Comment by aadcg** [comment]
Looks good to me!

File: source/manual.lisp [hunk 2]
```diff
@@ -228,32 +228,31 @@ all hosts being blocked, execute command " (:code "describe-variable") ", choose
...
          (:li "Automatically, by setting "
               (:nxref :slot 'prompt-on-mode-toggle-p :class-name 'modable-buffer)
               " to non-nil (refer to the "
               (:a :href "#configuration" "configuration section") " for help)."))
         (:p "All rules are stored at " (:code rules-file) ", which " (:u "is
-intended for read and write purposes")". You can find instructions at the top of
-mentioned file. The gist is that rules are mere Lisp lists which start with a
-condition that checks the URL. When conditions are met, modes are
-toggled. Besides user-defined conditions, the following are often useful: "
+intended for read and write purposes")". You can find instructions at the top of it.
```
Comment (Ambrevar) [inline source/manual.lisp:252]: While we are at it, "intended for read and write purposes" is hardly understandable.

I'd change it to "is meant to be human-readable and human-writable".

File: source/manual.lisp (hunk 1 above)
Comment (aadcg) [reply source/manual.lisp:242]: @createyourpersonalaccount could you please answer @aartaka

**Comment by aadcg** [comment]
Waiting on @createyourpersonalaccount to address the comments left by @aartaka.

**Comment by aadcg** [comment]
@aartaka could you please make the changes you suggest so that we merge this one?

**Comment by aartaka** [comment]
@aadcg, done in 9dc3354f7f8f2eb1e3a607adb8bf15ec29250a29.

**Comment by aadcg** [comment]
Merged via 162e8f040 and 46d21bba9.

Thanks.

**Comment by aartaka** [comment]
Cool, thanks for merging :D

**Pull request closed without merging**